cirras-underwriting-war/cirras-underwriting-war/src/main/angular/src/app/components/inventory/forage/forage-inventory.component.ts

```diff
@@ -453,6 +513,16 @@ export class ForageInventoryComponent extends BaseComponent implements OnChanges
     }
   }
 
+  clearPlantInsurability(fieldIndex, plantingIndex, invSeededIndex) {
```

Comment:
Maybe I don't understand this method correctly but shouldn't it check if the selected plant insurability is still allowed instead of just removing the selection?